calicoctl/commands/node/status.go

```diff
@@ -129,7 +136,19 @@ func drawTable(birdOut string) {
 
 		if s != "" {
 			col := []string{}
-			fields := strings.Fields(line)[3:6]
+			f := strings.Fields(line)[3:]
```

Comment:
OK - can you at least move the comment to the top of this section (since all of this is related to that comment)?

I'm worried that, as it is, this code is relatively hard to work through, not being the one who wrote it.